File: Hero/Models/UnderwritingDistributionEmail.cs
using System;
using System.Collections.Generic;

namespace Hero.Models
{
    public class UnderwritingDistributionEmail
    {
        public Guid QuoteUid { get; set; }
        public int QuoteId { get; set; }
        public bool IsPublished { get; set; }
        public bool IsBindable { get; set; }
        public int WordingVersionId { get; set; }
        public string CountryIsoCode { get; set; }
        public string StateProvinceCode { get; set; }
        public string ProductCode { get; set; }
        public Email Email { get; set; }
        public List<int> QuoteIds { get; set; }
        public string PolicyNumber { get; set; }
        public int BrokerId { get; set; }
        public Guid ClientUid { get; set; }
        public string ContactEmail => Email?.Sender?.Email;
    }
}

File: Hero/ClientApp/src/app/models/underwriting-distribution-email.ts
import { Email } from '@app/models';

export class UnderwritingDistributionEmail {
    public email: Email;
    public productCode: string;
    public countryIsoCode: string;
    public stateProvinceCode?: string;
    public wordingVersionId: number;
    public isPublished: boolean;
    public isBindable: boolean;
    public quoteUid: string;
    public quoteId: number;
    public quoteIds?: number[];
    public policyNumber?:string;
    public clientUid?: string;
    public brokerId?: number;
}

